.claude/skills/reap/SKILL.md

```
@@ -0,0 +1,89 @@
+mise run reap -- --pattern 'fnox export'                      # dry run: plan only
+mise run reap -- --pattern 'shims/git' --min-age 3600         # ignore anything under 1h
+mise run reap -- --pattern 'fnox export --format json' --kill # signals
+mise run reap -- --pattern '.*worker.*' --full-match --kill   # whole argv must match
+```
+
+`--kill` is the only thing that signals anything. Without it you get the plan
+and an exit code, which is what makes this safe to run first and think second.
+
+## Read the plan before you add `--kill`
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)